src/tree/SubscriptionTreeItem.ts

```diff
@@ -71,7 +71,7 @@ export class SubscriptionTreeItem extends SubscriptionTreeItemBase {
     public async createChildImpl(context: ICreateChildImplContext): Promise<AzureTreeItem> {
         const wizardContext: IVirtualMachineWizardContext = Object.assign(context, this.root, {
             addressPrefix: '10.1.0.0/24',
-            size: 'Standard_D2s_v3'
+            size: <ComputeManagementModels.VirtualMachineSizeTypes>'Standard_D2s_v3'
```

Comment:
Not sure why, but this is the only type that I have to cast.  Even though Standard_D2s_v3 is definitely a string in the type defined.

Reply:
Yeah that is weird. If you want something slightly more type-safe, you could do:
```
const size: ComputeManagementModels.VirtualMachineSizeTypes = 'Standard_D2s_v3';
const wizardContext: IVirtualMachineWizardContext = Object.assign(context, this.root, { addressPrefix: '10.1.0.0/24', size });
```